api/types/plugin.go

```diff
@@ -257,6 +282,14 @@ func (s *PluginSlackAccessSettings) CheckAndSetDefaults() error {
 	return nil
 }
 
+// CheckAndSetDefaults validates and set the default values
+func (s *PluginOpsgenieAccessSettings) CheckAndSetDefaults() error {
+	if s.Addr == "" {
```

Comment:
If this is Opsgenie API endpoint, can this field be renamed to `APIEndpoint`? `Addr` is too generic.